crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -364,6 +368,7 @@ impl From<Entity> for Identifier {
     }
 }
 
+#[cfg(feature = "serde")]
 impl Serialize for Entity {
```

Comment:
we can have serde::Serialize here and remove the feature import

Reply:
They're still required for reflecting the traits.